regolith/helpers/l_grantshelper.py

```diff
@@ -74,18 +74,13 @@ def sout(self):
 
         grants_time_info = []
         for i in grants:
-            start, end = '', ''
-            if i.get('begin_year') and i.get('end_year'):
-                times = get_dates(i)
-                start = times['begin_date']
-                end = times['end_date']
-                grants_time_info.append([i, start, end, end])
-            else:
-                # Creating fake dates with extreme values for grants that do not have an end date for sorting
-                grants_time_info.append([i, start, end, dt.date(3000, 1, 1)])
+            times = get_dates(i)
+            start = times['begin_date']
+            end = times['end_date']
+            grants_time_info.append([i, start, end])
```

Comment:
This is all ok, but I still don't get why you want to create a new list like this.  Why not just:
```
     for g in grants:
            print("{}, awardnr: {}, acctn: {}, {} to {}".format(
                                                                                  g.get('alias', ''),
                                                                                  g.get('awardnr', ''), 
                                                                                  get_dates(g).get('begin_date'),
                                                                                  get_dates(g).get('end_date')
                                                                                  )
```

Reply:
Because I need to have them sorted before I print so I first get the dates to sort the grants, and then store the dates so I don't have to call the function again when I print them

Reply:
If you would prefer it I can just call the function twice